Source/AwsCommonRuntimeKit/crt/Atomic.swift

```diff
@@ -0,0 +1,13 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+import AwsCCommon
+
+struct Atomic<I> {
```

Comment:
Hilarious. We don't even access the atomics in Swift. This is something I'll fix later when we move to the FFI library, this is unnecessarily part of the API in use here. Not Swift's problem.

Reply:
yep its the c atomic type I need access to so I can pass it but I agree you can abstract that away!